### PR TITLE
Updates to documentation

### PR DESCRIPTION
**Pull request by shyjaprabhu**
## Changes proposed in this pull request

(Fill in the changes here)

## Patch testing

(Fill in the details about how this patch was tested)

## ReleaseNotes.txt changes

(Does this change require an entry in ReleaseNotes.txt? If yes, has it been added to it?)

## Other PRs 

(Does this change require changes in other projects- store, spark, spark-jobserver, aqp? Add the links of PR of the other subprojects that are related to this change)

### CODE REVIEW

**Comment by dhavalstr** [comment]
Can one of the admins verify this patch?

File: docs/install/system_requirements.md [hunk 1]
```diff
@@ -18,15 +18,16 @@ SnappyData turns Apache Spark into a mission-critical, elastic scalable in-memor
 
 | Operating System| Version |
 |--------|--------|
-|Red Hat Enterprise Linux|RHEL 6.0 and 7.0|
-|Ubuntu|Ubuntu Server 14.04 and later|
-|CentOS|CentOS 6, 7|
+|Red Hat Enterprise Linux|- RHEL 6.0 </p> - RHEL 7.0 (Recommended Kernel version: 3.10.0-693.2.2.el7.x86_64)|
```
Comment (sumwale) [inline docs/install/system_requirements.md]: change to "Minimum recommended kernet version: ...". Also add JIRA issues numbers below in next line to clarify the reason for the recommendation.

File: docs/install/system_requirements.md [hunk 2]
```diff
@@ -18,15 +18,16 @@ SnappyData turns Apache Spark into a mission-critical, elastic scalable in-memor
...
+|Ubuntu|Ubuntu Server 14.04 and later||
+|CentOS|CentOS 6, 7 (Recommended Kernel version: 3.10.0-693.2.2.el7.x86_64)|
```
Comment (sumwale) [inline docs/install/system_requirements.md]: Same as previous comment.

File: docs/install/system_requirements.md [hunk 3]
```diff
@@ -18,15 +18,16 @@ SnappyData turns Apache Spark into a mission-critical, elastic scalable in-memor
...
+
 
 
 ## Host Machine Requirements
 Requirements for each host:
 
-* A supported [Java SE installation](http://www.oracle.com/technetwork/java/javase/downloads). SnappyData does not support Java SE 9.
+* A supported [Java SE 8u151/ 8u152 or earlier installation](http://www.oracle.com/technetwork/java/javase/downloads). We recommend using Java version: 1.8.0_144.
```
Comment (sumwale) [inline docs/install/system_requirements.md]: Change to something like "A supported [Java SE installation](...). Minimum version recommended is 1.8.0_144 which is what the product is currently tested with.

File: docs/release_notes/known_issues.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ The following key issues have been registered as bugs in the SnappyData bug trac
 <col width="25%" />
 <col width="25%" />
 <col width="25%" />
-<col width="25%" />
+<col width="25%" />
```
Comment (sumwale) [inline docs/release_notes/known_issues.md]: unnecessary space at end got added

**Pull request merged**